website_event_sale_b2x_alt_price/README.rst

```diff
@@ -56,7 +56,7 @@ Known issues / Roadmap
    scenarios. This is Odoo's bug and not this module's. It has been
    notified as OPW-2518694 but it doesn't look like it's going to be
    fixed any soon. In v16, event price discounts from pricelist don't
-   even work and for v17 there's an ongoing refactoring work.
+   even work. This issue has been resolved in Odoo v17.
```

Comment:
Then just remove the issue :sweat_smile:

Reply:
I have left it, changing the part where in v17 it is fixed since the module is not migrated to v16 yet. Although the most normal thing is that it goes from 15 to 16, it could be that whoever decides to migrate it will migrate it from 17 to 16. It is possible to leave historical, isn't it?

Reply:
I'd better remove it. Let's look for the future :smile:

Reply:
Done